splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/SpliceBaseOperation.java

```diff
@@ -608,11 +609,19 @@ a triggered statement(GenericTriggerExecutor.compile(), in those cases, we want
                 stmtForLogging = ps.getSourceTxt();
         }
 
+        UUID parentId = null;
+
+        try {
+            parentId = getCurrentTransaction().getParentQueryIdForTrigger();
+        } catch (Exception ignored) {}
+
```

Comment:
Log a warning message for the error

Reply:
I am logging the warning message in case getParentQueryIdForTrigger() results in one. Functino getCurrentTransaction may throw standardException, which I ignore it, as `logExecutionStart` originally did not call `getCurrentTransaction`